src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -1097,8 +1097,10 @@ ITypeSymbol IMethodSymbol.ReceiverType
             }
         }
 
-        // https://github.com/dotnet/roslyn/issues/35034: Implement
-        CodeAnalysis.NullableAnnotation IMethodSymbol.ReceiverNullableAnnotation => default;
+        CodeAnalysis.NullableAnnotation IMethodSymbol.ReceiverNullableAnnotation => ReceiverNullableAnnotation;
+
+        protected virtual CodeAnalysis.NullableAnnotation ReceiverNullableAnnotation =>
+            IsStatic ? CodeAnalysis.NullableAnnotation.NotApplicable : CodeAnalysis.NullableAnnotation.NotAnnotated;
```

Comment:
>CodeAnalysis.NullableAnnotation.NotAnnotated [](start = 71, length = 44)

Should we always return ```NotApplicable``` unless the method is a reduced extension method?

Reply:
@cston and I debated this a bit yesterday afternoon, and I ended up deciding that it felt more correct to have it be `NotAnnotated` unless the method didn't have a receiver. We warn when you invoke an instance method on a not null receiver, and it feels like the public API should reflect this fact.

---
In reply to: [277793575](https://github.com/dotnet/roslyn/pull/35191#discussion_r277793575) [](ancestors = 277793575)